.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: ruby
+rvm:
+  - "1.9.3"
```

Comment:
@Fryguy , @simon3z 
I think we'll have to limit the ruby version for this gem to be  2.0 and up (not 1.9.3) due to a change introduced in this pr:
https://github.com/abonas/kubeclient/pull/05/files#diff-78cbcd17c7ae3f02081ec1ce253bb73bR14
ruby 1.9.3 assumes path, not uri and hence running with ruby 1.9.3 fails with "NoMethodError: undefined method `empty?' for #<URI::HTTP:0x00000003054da0>"
see the diffs:
http://ruby-doc.org//stdlib-1.9.3/libdoc/net/http/rdoc/Net/HTTPGenericRequest.html#method-c-new
http://ruby-doc.org//stdlib-2.0/libdoc/net/http/rdoc/Net/HTTPGenericRequest.html#method-c-new